src/test/resources/jsonld/BT000041593.json

```diff
@@ -3,47 +3,36 @@
   "contribution" : [ {
     "type" : [ "Contribution" ],
     "agent" : {
-      "id" : "https://d-nb.info/gnd/10880724X",
       "type" : [ "Person" ],
-      "label" : "Altekamp, Heinrich",
-      "gndIdentifier" : "10880724X"
+      "label" : "Altekamp, Heinrich"
     },
     "role" : {
       "id" : "http://id.loc.gov/vocabulary/relators/aut",
       "label" : "Autor/in"
     }
   } ],
-  "coverage" : [ "Coesfeld (Kreis) | 97" ],
   "bibliographicCitation" : "Wirtschaftsspiegel. - 38 (1983) H. 7, S. 24-25 : 1 Abb.",
   "medium" : [ {
     "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010",
     "label" : "Print"
   } ],
   "spatial" : [ {
-    "id" : "https://nwbib.de/spatial#N97",
+    "id" : "https://nwbib.de/spatial#Q6210",
     "type" : [ "Concept" ],
     "source" : {
       "id" : "https://nwbib.de/spatial",
       "label" : "Raumsystematik der Nordrhein-Westfälischen Bibliographie"
     },
-    "notation" : "97",
-    "label" : "Kreise"
-  }, {
+    "label" : "Kreis Coesfeld",
+    "notation" : "05558",
     "focus" : {
       "id" : "http://www.wikidata.org/entity/Q6210",
+      "type" : [ "http://www.wikidata.org/entity/Q106658" ],
       "geo" : {
-        "lat" : 51.866666666667,
-        "lon" : 7.3833333333333
+        "lat" : "51.866666666",
+        "lon" : "7.383333333"
       },
-      "type" : [ "http://www.wikidata.org/entity/Q106658" ]
-    },
-    "id" : "https://nwbib.de/spatial#Q6210",
-    "type" : [ "Concept" ],
-    "label" : "Kreis Coesfeld",
-    "notation" : "05558",
-    "source" : {
-      "id" : "https://nwbib.de/spatial",
-      "label" : "Raumsystematik der Nordrhein-Westfälischen Bibliographie"
```

Comment:
Looks like we're losing the `source` field here.

Reply:
Seems also an effect of updating the data - the "new" data looks already exactly as the old one (curl -L  "accept: application/ld+json" 'http://lobid.org/resources/BT000041593#!' ).